configs/kubernetes-rules-configmap.yaml

```diff
@@ -6,6 +6,9 @@ data:
     skip-source-branches:
     - release-1.5
     - release-1.6
+    recursive-delete-patterns:
+    - BUILD
```

Comment:
might want to also catch `BUILD.bazel` files.

Reply:
That's only at the kubernetes root, isn't it?

Reply:
I mean: not in staging/src/k8s.io/*

Reply:
right, we generally use `BUILD` in the kubernetes repo, though apparently `BUILD.bazel` is now preferred. I vaguely intend to switch at some point, but if/when I do that I guess I can also update the bot at the same time.

Reply:
Then let's add it. It does not hurt.

Reply:
done